### PR TITLE
A fix to the Makefiles to account for an F* change

### PR DESCRIPTION
**Pull request by nikswamy**
See https://github.com/FStarLang/FStar/pull/2389

### CODE REVIEW

**Comment by nikswamy** [comment]
Let's merge this only after the F* PR is merged

**Review by beurdouche — APPROVED** [review]

**Review by polubelova — APPROVED** [review]
thanks!

**Comment by msprotz** [comment]
Looks like this was merged on the F* side a few hours ago. Thanks for the fix!

**Pull request merged**
